src/geckomat/model_adapter/ModelAdapter.m

```diff
@@ -18,7 +18,7 @@
         function model = manualModifications(obj,model) %default is to do nothing
```

Comment:
This line is not need any more, can be removed since is an artifact of the v2. Initial try to move/refactor `manualModifications.m`